packages/spark-core/components/toggle.js

```diff
@@ -46,6 +47,23 @@ const handleToggleClick = (toggleContent, toggleIcon, element) => {
   toggleAriaExpanded(element);
 };
 
+const handleToggleChevronIcon = (document, element) => {
+  // If the element contains 'sprk-c-Accordion__summary--open',
+  // set xlink:href attribute to be the filled icon and add a class to change
+  // styles.
+  const sprkChevronCircle = element.querySelector('use');
+  if (element.classList.contains('sprk-c-Accordion__summary--open')) {
+    const circle = document.querySelector('.sprk-SvgChevronCircleFilled--circle');
```

Comment:
Hey @janinavelasco9, wanted to give a bit of feedback before you got too far into unit tests. You'll want to do the query selection inside the bindToggleUIEvents. That away, you already have access to the accordion element and aren't running a query on the entire document again. So, you can more the const circle... and const arrow... into there. That will make the unit tests much more easy as well. Hope thats helpful.

Reply:
You might be able to utilize the toggleIconCSS function for adding classes as well. Nice job so far and thanks again for helping out!

Reply:
Yeahhh I didn’t push it up yet but I called that function in the click event in the bindtoggleuievent. But that’s a good idea to check out the toggle iconcss function. I’ll play around with that Monday. But for sureeee anything else you see needs work, let me know :) learning a lot.

Reply:
Yay! That’s great to hear. 😃